Gemfile

```diff
@@ -14,6 +14,9 @@ gem 'simplecov', '~> 0.10'
 gem 'test-queue'
 gem 'yard', '~> 0.9'
 
+gem 'rubocop-node_pattern',
+    git: 'git://github.com/jonatas/rubocop-node_pattern.git'
```

Comment:
Soon we migrate the gem to the proper repo, we can also publish as a gem and remove the git reference.

Reply:
Did you consider keeping git history using filter-branch feature for those files?

https://help.github.com/articles/splitting-a-subfolder-out-into-a-new-repository/

Reply:
That  is pretty cool @wevtimoteo, I'll use the tool to keep the git history intact 👍

Reply:
If you need some help with that I'll happy to pair with you!

Reply:
Done in https://github.com/jonatas/rubocop-astx/tree/move-ast-keeping-history